ts/features/bonus/siciliaVola/components/CheckResidenceComponent.tsx

```diff
@@ -56,23 +72,29 @@ const CheckResidenceComponent = (props: Props): React.ReactElement => {
       >
```

Comment:
```suggestion
      >
```
This `ref` is useless. My fault, I insert it in all the screens when I did the stubs story.
Can you remove it please?